src/config.ts

```diff
@@ -2,9 +2,15 @@ import GenericConfig from './platform/generic';
 import VercelConfig from './platform/vercel';
 import NetlifyConfig from './platform/netlify';
 
+declare global {
+  var Cloudflare: string;
```

Comment:
```suggestion
  var WebSocketPair: any; // Non-standard global only used on Cloudflare: https://developers.cloudflare.com/workers/runtime-apis/websockets
```

Sorry, I don't think there's a global `Cloudflare` object—best guess right now is checking for this non-standard global.

Reply:
Hi :wave:

Cloudflare Workers, and Pages Functions define `navigator.userAgent` as `'Cloudflare-Workers'`: https://developers.cloudflare.com/workers/runtime-apis/web-standards/#navigatoruseragent

This is probably the most future-proof way to detect running in Workers/Pages.

Reply:
Oh, that's cool—thanks @DaniFoldi 🙏